### PR TITLE
[Python] Update tensorflow doc

### PR DESCRIPTION
**Pull request by riteshghorse**
Add documentation for using built-in TensorFlow model handlers. 
[Preview](http://apache-beam-website-pull-requests.storage.googleapis.com/25954/documentation/ml/about-ml/index.html)
image

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by riteshghorse** [comment]
Run Whitespace PreCommit

**Comment by riteshghorse** [comment]
R: @AnandInguva @damccorm

File: website/www/site/content/en/documentation/ml/about-ml.md [hunk 1]
```diff
@@ -112,14 +112,19 @@ that illustrates running Scikit-learn models with Apache Beam.
...
 
-To use TensorFlow with the RunInference API, you need to do the following:
-
-* Use `tfx_bsl` version 1.10.0 or later.
-* Create a model handler using `tfx_bsl.public.beam.run_inference.CreateModelHandler()`.
-* Use the model handler with the [`apache_beam.ml.inference.base.RunInference`](/releases/pydoc/current/apache_beam.ml.inference.base.html) transform.
-
-See [this notebook](https://github.com/apache/beam/blob/master/examples/notebooks/beam-ml/run_inference_tensorflow.ipynb)
-that illustrates running TensorFlow models with Apache Beam and tfx-bsl.
+To use TensorFlow with the RunInference API, you have two options:
```
Comment (damccorm) [inline website/www/site/content/en/documentation/ml/about-ml.md:115]: I think we need to provide reasons on why you would use each. Specifically: the beam model handlers accept numpy/tensors, the tfx_bsl one requires tf.examples

Comment (damccorm) [reply website/www/site/content/en/documentation/ml/about-ml.md:115]: Same comment applies both places

**Review by damccorm — COMMENTED** [review]
Just had one small comment - thanks!

**Comment by riteshghorse** [comment]
Check finished but not updated here, updated the PR description

**Review by damccorm — APPROVED** [review]

**Pull request merged**